### DISCUSSION TITLE
Facility 3200 - What the hell is this?

### DISCUSSION THREAD

**Discussion by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#4150] [Q&A] [answered]
While finding other air defence stuff, I came across this unit in DB3k, a combined SA-11, SA-6 and SA-8 battery for Serbia. It has no sensors for the SA-11 and SA-8 TELARs and no reloads for the SA-11 either. Before mindlessly putting a ticket in to add the sensors I tried to find something about this rather odd operational concept, but I couldn't get anything. Serbia doesn't even have SA-11 or SA-8! Going by DB numbers it was added before this tracker was started and therefore possibly before the current team's time? Regardless, does anyone know what this is or what this is supposed to be?

**Comment by PygmalionOfCyprus** [comment, marked as answer]
God only knows. Either it was added to support someone's scenario because they couldn't be bothered to build it themselves, or they read about it in some obscure blog somewhere. Lot of junk in the DB like that, especially for "uncon" forces like the Serbs and Vietnamese. Or, perhaps it was added for some other country and got the wrong OperatorCountry?

Don't worry about it. We'll probably reassign it whenever we get to doing a nat review of those guys.
